Examples/ObjectiveC/CacheManagementExample.m

```diff
@@ -0,0 +1,171 @@
+#import "CacheManagementExample.h"
+@import Mapbox;
+
+NSString *const MBXExampleCacheManagement = @"CacheManagementExample";
+
+@interface CacheManagementExample () <MGLMapViewDelegate>
```

Comment:
I might be missing something, but I don’t see that this needs to conform to `MGLMapViewDelegate`.